doc/guide/Makefile-guide.am

```diff
@@ -98,6 +98,11 @@ dist/guide/html/index.html: $(GUIDE_DOCBOOK) $(GUIDE_INCLUDES) $(man_MANS) $(GUI
 		LANG=en_US.UTF-8 $(XMLTO) html -m $(srcdir)/doc/guide/gtk-doc.xsl -o dist/guide/html/ \
```

Comment:
nitpick: there is no need to separate the four commands with &&, as make already runs all the commands in `set -e` style. Then the double indentation could be dropped (after the `MKDIR_P`).